README.md

```diff
@@ -1,46 +1,148 @@
+ - The assignee defines solved and moves the issue from [In Progress] to [Done].
+ - The [PO] checks the [Done] issues regularly and informs the `Stakeholder` to check/test the solution for the issue:
+   - This can be delegated to/ done by the issue assignee.
+
+### Prior to the Sprint Planning / Review meeting
+1. All: Go through all issues that are assigned to you
+   1. in [Sprint Backlog]
+      1. are you working on an issue? --> move it to [In Progress]
+   1. in [In Progress]
+      1. is an issue solved? --> move it to Done
+   1. in [Done]
+      1. talk to the [PO] or `Stakeholder` (creator of the issue in general) of the issue, if the provided solution is accepted [Approved by Stakeholder]
+ 1. All: Go through all issues that are not assigned to you
+    1. in [Done]
+       1. +1 or -1 any issue
+    1. in [Product Backlog]
+       1. understand the issues here
+       1. do they fit your skills/ goals
+       1. be prepared to talk about them/ discuss complexity/ estimates
+ 1. [PO]
+    1. [Stakeholder Backlog] see above: check priority and well defined; assign or label or/and move to [Product Backlog]
+    1. [Done] see above: inform `Stakeholder` about the solved issues and request 
+
+### Sprint
+The [Sprint] is the time period between Sprint Planning and Sprint Review.
+We will use a 2 weeks period.
+We combine the Sprint Planning and the Sprint Review into one meeting at the beginning of every other week.
+During the [Sprint] the assignees work on their issues that are in the [Sprint Backlog].
+As a rule of thumb, an issue needs to have 13 or less Story Points (see [Glossary] & [Story Points Estimates]) to be solvable in one [Sprint] (two weeks).
+Therefore an associate can have only 13 or less Story Points alltogether in one [Sprint].
+These are just general guidelines. Over time each team can find its own rules and time estimates ([Velocity])
+
+
+### Approved by Stakeholder
+tbd
+
+#### During the [Sprint]
+tbd
+
+### Redefine Issues & Epics
+Goal of [Redefine Issues & Epics] is to understand issues better if needed and split issues bigger than 13 Story Points (= Epics) into smaller issues (<= 13 Story Points).
+This will lead to issues that can be direct worked on (allow them into the [Product Backlog])
+
+tbd: when, who and how - hands on
+
+---
+
+## Synchronous work
+
+### Sprint Planning / Review meeting
+Goal of the Sprint
+
+#### Review 
+#### Sprint Planning
+
+#### During the meeting
 
 1. issues in the `Done` column with a -1 are discussed and all other issues are closed out
 2. issues in the `Product Backlog` are assigned and moved to the `Sprint Backlog`
 
+#### Sprint Backlog
+
 ## Product Owner
+The [Product Owner] is repsonsible for:
+ - Priorizing issues: Move it from [Stakeholder Backlog] to [Product Backlog]
+ - Well defined: Approve an issue or Request a better defintion --> set `triage/accepted` label
+ - Check or delegate the [Approval by Stakeholder]
 
 @goern and @durandom are product owners
```

Comment:
I don't think @mentions work in the markdown, maybe provide a link?

Reply:
```suggestion
[@goern](https://github.com/goern) and [@durandom](https://github.com/durandom) are product owners
```